### ISSUE TITLE
Counters time and iterations/time values get erroneously updated even after Counter is closed

### ISSUE THREAD

**Issue by bedaberner** [Rockhopper-Technologies/enlighten#62]
**Describe the bug**
The Time and iteration/time values get updated, even after the counter is closed.
This seems to happen when a new counter is introduced.

This is pretty annoying if you are interested in how long substeps of a long process actually took.

**To Reproduce**

```
import enlighten
import time


manager = enlighten.get_manager()

A = manager.counter(total=10, desc='A', unit='iter')
for num in range(10):
    time.sleep(0.1)  # Simulate work
    A.update()
A.close()
time.sleep(5)
B = manager.counter(total=10, desc='B', unit='iter')

manager.stop()

```
In this example you can see that the elapsed time for A changes as soon as B is created. In the end it will show 6s instead of the 1s it actually took.
Funnily enough, the issue is also present in demo.py and you can see it in the GIF on the documentation landing page

**Environment (please complete the following information):**
 - Enlighten Version: 1.11.2
 - OS and version: Ubuntu 20.04.6 Python 3.8.10
 - Console application: VS Code Terminal
 - Special Conditions: -

**Comment by avylove** [comment]
Good catch! Thanks for reporting!

I'm currently working on the next release and should be able to get a fix in. I'm thinking a new instance variable is needed which tracks when the ``count`` was updated and change ``count`` to a property so it gets set regardless of how ``count`` is updated. 

Previously ``last_updated`` was used to track this, but that really just tracks when the last refresh was, so the implementation was incomplete.

**Comment by avylove** [comment]
Fixed in [1.12.0](https://github.com/Rockhopper-Technologies/enlighten/releases/tag/1.12.0)

**Comment by bedaberner** [comment]
awesome, thanks for fixing this so fast!
